### DISCUSSION TITLE
Thank you very much!! first calls got blocked today!

### DISCUSSION THREAD

**Discussion by deltahoch3** [haumacher/phoneblock#32] [General]
just wanted to shot out a big thank you, my mothers phoneline gets "attacked" by 1-2 calls per day, but today it was silent!
keep it up and thank you alot!!

**Comment by haumacher** [comment]
You are welcome! Thanks for your acknowledgment. You can help [PhoneBlock](https://phoneblock.net/phoneblock/) to get more popupar by posting links to the website, telling your friends and family about the project and help them to setup the blocklist in their VOIP routers.

**Reply by deltahoch3** [reply]
howdy mr. haumacher,
just added my first spam-number :)

but sofar it work pretty damn good! 
![2024-01-11_13-25-42_screenshot](https://github.com/haumacher/phoneblock/assets/110137184/b17be67d-6f35-4e6b-9dcd-dfa3412dcb2a)

sofar i have the list with 1000 entries running. i got an fritzbox 7490,
maybe  you could add a section in your wiki / website, where users can post their router with maximum addressbook size?
what happens if i choose a "too-big" addressbook, will this crash/"break" my fritzbox?

keep it up and THANKS again!

**Reply by deltahoch3** [reply]
fyi: https://www.reddit.com/r/fritzbox/comments/1940txh/phoneblock_end_of_spamcalls_aka_telefonterror/

i just posted it on the fritzbox subreddit, not  sure if i described it right... ;)

**Reply by haumacher** [reply]
Thank you for the mention - some other people have found PhoneBlock through your post!
